improver.py

```diff
@@ -0,0 +1,75 @@
+# Written by Goncalopp and RedFantom
+# Thranta Squadron GSF CombatLog Parser, Copyright (C) 2016 by Goncalopp and RedFantom
+# All additions are under the copyright of their respective authors
+# For license see LICENSE
+from PIL import ImageEnhance, Image, ImageOps
+import files
+import numpy
+import cv2
+
+
+class ImageImprover(object):
+    """
+    Improver class that provides a wrap around the ImageEnhance classes from PIL/pillow
+    The Improved image can be accessed through object.image, this is a PIL/pillow RGB
+    format image, that is NOT compatible with the cv2 based files.ImageFile class!
```

Comment:
What I meant is that the internal state (`self.image`) of ImageImprover is a PIL image.

It seems like this (`ImageImprover`) could more easily be done by two standalone functions (`enhance`, `crop`) that take the image as the argument and could then be used directly from `ImageFileImprover`. That would avoid the code duplication as well, and you wouldn't necessarily need to keep a PIL image at all, since you can always convert the input image from cv to PIL, and convert the output from PIL to cv before returning. It's probably not very efficient, but avoids ever exposing PIL images - what do you think? Would this work with your fork code so far?